.github/workflows/ci.yml

```diff
@@ -8,4 +8,65 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
-      - run: yarn && yarn lint
+        with:
+          node-version: '14'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Run lint
+        run: yarn lint
+
+  Test:
+    runs-on: ubuntu-latest
+    services:
+      verdaccio:
+        image: verdaccio/verdaccio:4.0.0
+        ports:
+          - 4873:4873
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: |
+          yarn install --frozen-lockfile
+          yarn --cwd fixtures/app install --frozen-lockfile
+
+      - name: Verdaccio login
+        run: |
+          npx npm-cli-adduser \
+            -u username \
+            -p pass \
+            -e username@email.com \
+            -r http://localhost:4873 || true
+
+      - name: Configure git
+        run: |
+          git config --global user.name 'Verdaccio User'
+          git config --global user.email 'username@email.com'
+
+      - name: Get changed packages
+        id: changed-packages
+        run: |
+          changed_packages=$(yarn -s lerna list --since | sed 's/$/\@0.0.0-verdaccio/' | xargs echo)
+
+          echo "::set-output name=PACKAGES::$changed_packages"
+
+      - name: Publish packages
+        run: |
+          yarn lerna publish \
+            '0.0.0-verdaccio' \
+            --registry http://localhost:4873 \
+            --no-git-tag-version \
+            --no-push \
+            --yes
+
+      - name: Install packages and run test
+        if: steps.changed-packages.outputs.PACKAGES
```

Comment:
Today the E2E tests are running over old version of the packages, maybe we should always publish the `0.0.0-verdaccio` versions locally for all packages to be sure the tests are using the locally version?

Reply:
yeah, I've made it so it only installs the packages that were actually changed, else it will use their current version published in NPM. I don't think it should make a difference whether we use the latest version on NPM or the version on the repo (if they weren't changed), right?

Reply:
maybe I should update the tag on `package.json` to instead of specifying the version to use the `latest` tag. wdyt?